apps/gateway-ui/src/languages/en.json

```diff
@@ -69,7 +69,7 @@
     "sort": "Sort"
   },
   "info-card": {
-    "amboss_node_link_text": "View on amboss.space",
+    "open-node-in-explorer": "Open in explorer",
```

Comment:
Not important, but we could change this to `View on {{origin}}` and update the `t` calls to include the origin of the link they pulled together. Not really worth holding up the PR, but ideally we avoid deviating from design wherever possible.

Reply:
Interpolation with origin results in this madness `"View on https:&#x2F;&#x2F;mempool.space"`. Went with `View on {{host}}`